### PR TITLE
Version/3.14

### PR DESCRIPTION
**Pull request by gaborznagy**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,87 +1,65 @@
...
-
-# Credits
+3.14.1
+
+<!-- Fri, 23 Feb 2018 09:08:05 +0100 -->
+
+## Features
+
+ * Password protected ssl keys (#1888)
+ * Add OpenBSD modul to system() source (#1875)
```
Comment (MrAnno) [inline NEWS.md]: *module

File: NEWS.md [hunk 2]
```diff
@@ -1,87 +1,65 @@
...
+ * Fix a memory leak in cfg-lexer (#1843)
+ * Fix some issues found by pylint in python module (#1881, #1830)
+ * Fix a crash due to a race condition in kv-parser() (#1871)
+ * Fix a crash due to a race condition in file() destination (#1858)
+ * Fix deprecated API usage in python module tests (#1829)
+ * Fix a race condition in internal() source (#1815)
+ * Fix a locale issue in merge-grammar python tool (#1868)
+ * Fix compile problems with autotools when '--disable-all-modules' used (#1853)
+ * Fix a file descriptor leak in persist-state (#1847)
+ * Fix a file descriptor lean in pseudofile() (#1846)
```
Comment (MrAnno) [inline NEWS.md]: *leak

**Comment by kira-syslogng** [comment]
Build FAILURE, the tests were executed on test branch: master and test suite: functions

**Comment by gaborznagy** [comment]
Fixed comments

**Comment by kira-syslogng** [comment]
Build FAILURE, the tests were executed on test branch: master and test suite: functions

**Comment by lbudai** [comment]
@kira-syslogng retest this please;

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Pull request merged**